observability/cockpit/how-to/create-token.mdx

```diff
@@ -0,0 +1,54 @@
+---
+meta: How to create a token
+  title: How to create a token
+  description: This page shows you how to create a token
+content:
+  h1: How to create a token
+  paragraph: This page shows you how to create a token
+categories:
+  - observability
+dates:
+  validation: 2022-10-17
+  posted: 2022-10-30
+---
+
+This page shows you how to create [tokens](/observability/cockpit/concepts/#tokens), to grant other cloud providers’ services and platforms access to your [Cockpit](/observability/cockpit/concepts/#cockpit).
+
+<Message type="requirement">
+
+ - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+ - You have activated your [Cockpit](/observability/cockpit/how-to/activate-cockpit)
+
+</Message>
+
+
+1. Click **Cockpit** in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
+
+2. Click the **Tokens** tab.
+
+3. Click **Generate new token** to create a token. A pop-up displays prompting you to create a token.
+
+4. Choose a name for your token or use the already-generated name.
+
+5. Select the permissions you wish to apply to your token.
```

Comment:
I believe this page would bear more value if each permission was explained as it may not be very straightforward.